code/modules/spells/general/lightning.dm

```diff
@@ -78,62 +77,39 @@
 	return temp
 
 /spell/lightning/process()
-	if(chargedkey) return //do not charge while we are gonna zap
+	if(currently_channeled) return //do not charge while we are gonna zap
```

Comment:
I can remove this entirely I think since the system doesn't take charge until you cast this way anyway. I don't think this was ever necessary except for lightning's multicast system.